backend/stakepoold/stakepool/stakepool.go

```diff
@@ -472,6 +477,51 @@ func (spd *Stakepoold) UpdateUserDataFromMySQL() error {
 	return nil
 }
 
+// Enable early detection of outages by periodically scraping dcrd and
+// dcrwallet, and exporting the results to a monitoring system.
+func (spd *Stakepoold) ExportPrometheusMetrics(ctx context.Context, wg *sync.WaitGroup, wait time.Duration, addr string) {
+	promConnectionCount := promauto.NewGauge(prometheus.GaugeOpts{
+		Name: "dcr_peer_connections",
+		Help: "Returns a count of connections from peers",
+	})
+	promTicketsLive := promauto.NewGauge(prometheus.GaugeOpts{
+		Name: "dcr_tickets_live",
+		Help: "Returns a count of live tickets",
+	})
+	promTicketsMissed := promauto.NewGauge(prometheus.GaugeOpts{
+		Name: "dcr_tickets_missed",
+		Help: "Returns a count of missed tickets",
+	})
+	promBlockHeight := promauto.NewGauge(prometheus.GaugeOpts{
+		Name: "dcr_block_height",
+		Help: "Returns the latest block height",
+	})
+
```

Comment:
need to wg.Add(1) here

Reply:
Yeesh, thanks. I had it in before but must have dropped it accidentally.